src/snippets/oauth2-code-flow.mdx

```diff
@@ -1,27 +1,23 @@
-If you are registering a new application, you must first save the configuration.
+If you are registering a new application, please save the initial configuration first.
 
-Once you have a saved application registration you may configure the OAuth2 code flow. 
+After saving, you can configure the OAuth2 Code Flow for this application by following the three steps:
 
-Open the application registration and configure it for the right OAuth2 flow:
-
-1. Enable OAuth2 code flow
-2. Copy the generated client secret. 
+1. Enable OAuth2 Code Flow.
+   ![OAuth2 client secret](./images/oauth2-enable.png)
+2. Copy the generated client secret. _Note that this is the only time you will be shown the actual value of the client secret. Criipto only stores this as a hashed value, which means you cannot retrieve the value once it has been generated and stored._
+   ![OAuth2 client secret](./images/oauth2-client-secret.png)
 3. Set the user info response strategy to `plainJson` to enable retrieval of plain JSON user information from the `/oauth2/userinfo` endpoint.
+   ![OAuth2 code flow](./images/oauth2-code-flow.png)
+   _Note that some libraries do not support the final `userinfo` request. In those cases, you will need to fetch the user data directly from the `token` endpoint as opposed to the `userinfo` endpoint._
 
-![OAuth2 code flow](./images/oauth2-code-flow.png)
-
-_Note that this is the only time you will be shown the actual value of the client secret_. Criipto only stores this as a hashed value, which means you cannot retrieve the value once it has been generated and stored.
-
-![OAuth2 client secret](./images/oauth2-client-secret.png)
+<br />
 
 <Highlight icon="file-lines">
 
-Some libraries do not support the final `userinfo` request. In those cases you will need to fetch the user data directly from the `token` endpoint as opposed to the `userinfo` endpoint. Do this by choosing the appropriate option as shown below.
-
-</Highlight>
-
-You may configure Criipto Verify  to retrieve the user information from  either the `userinfo` endpoint - the default option - or you may explicitly choose the `fromTokenEndpoint` in the user info response strategy instead:
-
-![User info response strategy](./images/userinfo-responsestrategy-fromtokenendpoint.png)
+Criipto Verify supports three modes for retrieving user information:
 
+- Plain JSON object (`plainJson`): The user information is fetched as a standard JSON object from the `userinfo` endpoint.
+- Signed JWT (`signedJwt`): The user information is fetched as a digitally signed JSON Web Token from the `userinfo` endpoint.
+- Directly from the `token` endpoint, embedded in the `id_token` (`fromTokenEndpoint`). _The `fromTokenEndpoint` flow is not standard, but can be useful if you are working with a product that does not call the `userinfo` endpoint._
```

Comment:
@mickhansen please see if you agree with mentioning all 3 options we currently have.